source/_posts/2023-08-01-php-core-roundup-15.md

```diff
@@ -0,0 +1,413 @@
+
+CI/CD platforms that use Docker images can use the PHP 8.2 docker images available with various base images. GitHub Actions can also make use of <code>[shivammathur/setup-php](https://github.com/shivammathur/setup-php)</code> action, which supports PHP 8.3 builds.
+
+## PHP Bug Fix Releases
+
+The PHP development team released two new versions in July 2023:
+
+**[PHP 8.2.8](https://www.php.net/archive/2023.php#2023-07-06-2)** and **[PHP 8.1.21](https://www.php.net/archive/2023.php#2023-07-06-3)**
+
+These releases include bug fixes across various components such as CLI, Core, Curl, DOM, Opcache, OpenSSL, PGSQL, Phar, SPL, and Standard.
+
+## Recent RFCs and Mailing List Discussions
+
+
+> Changes and improvements to PHP are discussed, reported, and voted on by the PHP Foundation Team, the PHP development team, and contributors. Bug reports are made to the PHP [issue tracker](https://github.com/php/php-src/issues), changes are discussed in [mailing lists](https://www.php.net/mailing-lists.php), minor code changes are proposed as [pull requests](https://github.com/php/php-src/issues), and major changes are discussed in detail and voted on as [PHP RFCs](https://wiki.php.net/rfc). [Documentation](https://github.com/php/doc-en/) and the [php.net website](https://github.com/php/web-php) changes are also discussed and improved at their relevant Git repositories on GitHub.
+
+<br>
+Hundreds of awesome PHP contributors put their efforts into improvements to the PHP code base, documentation, and the php.net website. Here is a summary of some changes made by the people behind PHP. Things marked with 💜 are done by the PHP Foundation team.
+
+## RFC Updates
+
+### In Voting: [PDO driver specific sub-classes](https://wiki.php.net/rfc/pdo_driver_specific_subclasses) by Danack
+
+PHP's PDO extension supports connecting to multiple database software by using different database drivers available. Some of these drivers include MySQL, PostgreSQL, and SQLite. Although most of these database support a common set of features, all of these databases have evolved to provide additional features that PDO _sometimes_ can support, but there is no easy way for a library to check for these capabilities without having to inspect the driver and their versions.
+
+This RFC proposes to create driver-specific sub classes of `\PDO`, so libraries and applications can easily indicate the database driver they expect, and make use of IDE autocompletion and proper error handling. 
+
+The vote finishes on August 3rd, and the tally is an unanimous "Yes" at the moment. 
```

Comment:
No, the vote finished on July 17th

Reply:
Brainfart, thanks a lot for the quick review and pointing this out. Updated.